src/lib/operators/get_table.hpp

```diff
@@ -2,20 +2,24 @@
 
 #include <memory>
 #include <string>
+#include <utility>
 #include <vector>
 
+#include "../storage/storage_manager.hpp"
 #include "abstract_operator.hpp"
 
 namespace opossum {
 
 // operator to retrieve a table from the StorageManager by specifying its name
 class GetTable : public AbstractOperator {
  public:
-  explicit GetTable(const std::string& name);
+  explicit GetTable(std::string table_name) : _name(std::move(table_name)) {}
 
-  const std::string& table_name() const;
+  [[nodiscard]] const std::string& table_name() const { return _name; }
```

Comment:
I wonder wether the `[[nodiscard]]` is actually needed. Because the sole intent for this is to return  _name, it won't ever be used for any side effects the return-value won't be discarded.